src/__tests__/observer.spec.js

```diff
@@ -60,119 +54,119 @@ describe('#getPooled', () => {
         expect(getPooled(defaultOptions)).toEqual(instance);
     });
 
-    test('new IntersectionObserverContainer returns a pooled IntersectionObserver instance', () => {
-        const observer = IntersectionObserverContainer.create(noop, defaultOptions);
+    test('createObserver returns a pooled IntersectionObserver instance', () => {
+        const observer = createObserver(defaultOptions);
         expect(observer).toEqual(getPooled(defaultOptions));
     });
 });
 
-describe('#observe', () => {
-    test('observing a React instance when observer is already in storage', () => {
-        const observer = IntersectionObserverContainer.create(noop, defaultOptions);
+describe('#observeElement', () => {
+    test('observing a React instance when observer is already in observerElementsMap', () => {
+        const observer = createObserver(defaultOptions);
         const spy = jest.spyOn(observer, 'observe');
         const targets = new Set();
-        storage.set(observer, targets);
+        observerElementsMap.set(observer, targets);
         const element = { target: { nodeType: 1, id: 1 }, observer };
-        IntersectionObserverContainer.observe(element);
+        observeElement(element);
 
         expect(Array.from(targets)[0]).toEqual(element);
         expect(spy).toBeCalled();
     });
 
-    test('observing a React instance when observer is not in storage yet', () => {
-        const observer = IntersectionObserverContainer.create(noop, defaultOptions);
+    test('observing a React instance when observer is not in observerElementsMap yet', () => {
+        const observer = createObserver(defaultOptions);
         const spy = jest.spyOn(observer, 'observe');
         const element = { target: { nodeType: 1, id: 2 }, observer };
-        IntersectionObserverContainer.observe(element);
-        const targets = Array.from(storage.get(observer));
+        observeElement(element);
+        const targets = Array.from(observerElementsMap.get(observer));
```

Comment:
Should we use `observerElementsMap.get(observer).values().next().value` instead?

Reply:
That would make sense in "real" code, in tests it only makes sense if it makes the test easier to understand.